.github/workflows/_reusable_run_tests.yml

```diff
@@ -42,7 +42,7 @@ env: # https://docs.fastlane.tools/getting-started/ios/setup/
 jobs:
   run-tests:
     name: Run tests
-    runs-on: ghcr.io/cirruslabs/macos-runner:sequoia
+    runs-on: ghcr.io/cirruslabs/macos-sequoia-xcode:16.4
```

Comment:
question: when you say lightweight do you mean it boot up quicker?

Reply:
yes let's revert it please, I would keep the same image as before, Cirrus-CI was recommended this as far as i remember